src/popup/pages/Dashboard.vue

```diff
@@ -29,7 +29,7 @@
 
         <template #buttons>
           <OpenTransferReceiveModalButton is-big />
-          <OpenTransferSendModalButton is-big />
+          <OpenTransferSendModalButton is-big :is-air-gap="isAirGap" />
```

Comment:
Probably this button doesn't require airgap flag property as the TransferSend modal can establish this on its own.